scalatags/shared/src/main/scala/scalatags/generic/Util.scala

```diff
@@ -43,12 +43,13 @@ trait Util[Builder, Output <: FragT, FragT] extends LowPriUtil[Builder, Output,
     /**
      * Converts the string to a [[Attr]]
      */
-    def attr = {
+    def attr: Attr = attr(None)
+    def attr(namespace: Option[Namespace]): Attr = {
```

Comment:
OK, but the same situation is with tags and that's the way original author did it, so maybe it would be nice to keep things consistent?

BTW I don't like `Option[Namespace]` in public interface. Having to always pass `Some(ns)` seems clunky to me. IMHO, a separate method, `nsAttr` (or something) would be better. Or even taking `Namespace` and using `null` to represent no namespace (assuming that `null` will never be explicitly passed by user but just used as default value).